lib/tasks/evm_application.rb

```diff
@@ -179,6 +179,10 @@ def self.deployment_status
     return "new_replica"    if MiqServer.my_server.nil?
     return "upgrade"        if context.needs_migration?
     "redeployment"
+  rescue PG::ConnectionBad, ActiveRecord::NoDatabaseError => err
+    raise unless err.message.match?(/database "vmdb_(development|production)" does not exist/)
```

Comment:
Not sure about being specific with development|production...I'd be fine with:

```suggestion
    raise unless err.message.match?(/database "[a-z_]+" does not exist/)
```